content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/autocomponents.md

```diff
@@ -26,7 +27,7 @@ The `Component` tag defines the name, namespace, include path, and override beha
 
 ### ComponentRelation
 
-A component relation (the `ComponentRelation` tag) describes how various components may be related to the component being described. 
+The `ComponentRelation` tag indicates a component relation, which describes how various components may relate to the component that's being described.
```

Comment:
This definition is sounds a bit circular, I dont think I would understand this without having to read to further about the different properties. I like concrete examples:

_The `ComponentRelation` tag indicates this component's relationship with other components; define if this component requires or is incompatible with sibling components on the same entity. For example, the NetworkCharacterComponent requires a NetworkTransformComponent on the same entity in order to properly function._